sdk/eventhub/event-hubs/src/managementClient.ts

```diff
@@ -296,6 +296,12 @@ export class ManagementClient extends LinkEntity {
         maxRetries: options.retryOptions && options.retryOptions.maxRetries,
         abortSignal: options.abortSignal,
         requestName: options.requestName,
+        timeoutInSeconds:
+          options.retryOptions == undefined ||
```

Comment:
For readability, pull this code out and have a variable/constant hold the value of the timeout